### ISSUE TITLE
add example how to use: requires_scope

### ISSUE THREAD

**Issue by copdips** [Azure-Samples/ms-identity-python-webapi-azurefunctions#4]
Could you please add example how to use `requires_scope()`:
https://github.com/Azure-Samples/ms-identity-python-webapi-azurefunctions/blob/83a67bd97d6a97f2094b3044a40879dbc1ef6feb/Function/secureFlaskApp/__init__.py#L127

**Comment by giuliohome** [comment]
Yes, I would ask the same, notice also that the access token must be produced with a scope/claim otherwise the `aud` of the access token won't be correct, see https://stackoverflow.com/questions/74886417/azure-authentication-access-token-returning-wrong-aud00000003-0000-0000-c000
